mbed-hal/spi_api.h

```diff
@@ -119,45 +177,34 @@ int  spi_master_write(spi_t *obj, int value);
 /** Checks if the specified SPI peripheral is in use
  *
  * @param[in] obj The SPI peripheral to check
- * @return non-zero if the peripheral is currently transmitting
+ * @return  rc  `SPI_RC_NOT_BUSY` or `SPI_RC_BUSY`
  */
-int  spi_busy(spi_t *obj);
-
-/** Get the module number
- *
- * @param[in] obj The SPI peripheral to check
- * @return The module number
- */
-uint8_t spi_get_module(spi_t *obj);
```

Comment:
You think there's no use for spi_get_module() function? The intention there was to get instance number of peripheral (something like SPI_MODULE_x).